model/Core/Classes/Actor.md

```diff
@@ -0,0 +1,20 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# Actor
+
+## Summary
+
+An individually-identified Person, Organization or SoftwareAgent.
+
+## Description
+
+Actor is a grouping of identifying characteristics unique to an individual person, organization or software agent.
+
+## Metadata
+
+- name: Actor
+- SubclassOf: Entity
+- Instantiability: Concrete
+
+## Properties
+
```

Comment:
This misses the newly introduced property?
According to the model.png, this should have a property `identifier` of type `Email`.

Issue: `identifier` is already defined in [ExternalIdentifier.md](https://github.com/spdx/spdx-3-model/blob/main/model/Core/Classes/ExternalIdentifier.md) with type `xsd:string`.

Double issue: The `identifier` property is missing from the [Properties](https://github.com/spdx/spdx-3-model/tree/main/model/Core/Properties) folder.

Reply:
The ExternalIdentifier property and the Agent/Actor property have different semantics and different syntax - Agent must be a local@domain identifier that uniquely identifies a single instance.  ExternalIdentifier can (and usually does) identify a class of things (such as a product) rather than an individual instance of that product.

It's not a problem for two Classes to have properties with the same name and different types (Email vs. unrestricted String).  But creating a Property file would require listing one type and ignoring the other.  Email, SemVer and NonNegativeInteger Classes don't have that problem.  Recommendation: do not create an `identifier` file in the Properties folder.

Reply:
[SemVer](https://github.com/spdx/spdx-3-model/tree/main/model/Core/Classes) is a Class.
The "specVersion" property of [CreationInfo](https://github.com/spdx/spdx-3-model/blob/main/model/Core/Classes/CreationInfo.md) is type SemVer.  Assume for the sake of argument that the property name is "version".

Now assume that you want [ExternalIdentifier](https://github.com/spdx/spdx-3-model/blob/main/model/Core/Classes/ExternalIdentifier.md) to have a "version" property, but that ExternalIdentifier versions are PURLs (or Integers, or anything - it doesn't have to make sense).  Add the version property to the ExternalIdentifier class and voila, it's represented.

Reply:
In general, we do not follow an object-oriented model where we have "classes" that have "properties". We define the ontology which includes Classes and Properties. Instances of more than one classes may have the same properties. For example both an instance of an `Element` and of a `CreationInfo` have a `comment`. And it goes the other way: if _something_ has an `algorithm` and a `hashValue`, then this something also is of class `Hash`.

Therefore we can not "add the version property to the ExternalIdentifier class" in your example.